flow/src/main/java/com/sequenceiq/flow/reactor/config/EventBusConfig.java

```diff
@@ -21,8 +21,14 @@
 @Configuration
 public class EventBusConfig {
 
-    @Value("${cb.eventbus.threadpool.core.size:50}")
-    private int eventBusThreadPoolSize;
+    @Value("${cb.eventbus.threadpool.core.size}")
+    private int eventBusThreadPoolCoreSize;
+
+    @Value("${cb.eventbus.threadpool.max.size:150}")
```

Comment:
I think we should add all of the eventbus's thread pool properties to the application.yml or none of them, but the mixed approach could be tricky if the pool size bigger then the max pool size :)

Reply:
removed default value from application.yml